swift/ql/src/consistency/PrintAstConsistency.ql

```diff
@@ -0,0 +1 @@
+import codeql.swift.printast.Consistency
```

Comment:
I think we should not include these queries in the `codeql/swift-queries` pack. They are not meant for users. Other languages seem to store them in separate packs `LANG/ql/consistency-queries`.

Reply:
makes sense!

Reply:
done